src/modals/view-issue-modal.tsx

```diff
@@ -157,12 +159,34 @@ export const ViewIssueModal = ({
 
       register({
         key: "o",
-        name: "Open in browser",
+        name: "Linked resources",
         handler: () => {
-          const issue = store.get(issueAtom);
-          if (issue) {
-            openIssueInBrowser(issue.key);
+          openModal("linkedResources", store.get(issueAtom)!.id);
+        },
+      });
+
+      register({
+        key: "y",
+        name: "Copy ticket number",
+        handler: () => {
+          clipboard.writeSync(store.get(issueAtom)!.key);
+        },
+      });
+
+      register({
+        key: "Y",
+        modifiers: ["shift"],
+        name: "Copy branch name",
+        handler: () => {
+          const issue = store.get(issueAtom)!;
+
+          if (!issue) {
+            return;
           }
+
+          const branchName = `${issue.key}-${hyphenatedSummary(issue.fields.summary)}`;
```

Comment:
This doesn't use the formatted branch name. Maybe we should extract these common handlers, with the `issue` as input.

`handler: () => copyFormattedBranchName(issue)`